src/smart-components/group/policy/policies.js

```diff
@@ -0,0 +1,104 @@
+import React, { Fragment, useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+import { expandable } from '@patternfly/react-table';
+import AddPolicyWizard from './add-policy/add-policy-wizard';
+import EditPolicy from './add-policy/edit-policy-modal';
+import { TableToolbarView } from '../../../presentational-components/shared/table-toolbar-view';
+import RemovePolicy from './../policy/add-policy/remove-policy-modal';
+import { createRows } from './policy-table-helpers';
+import { fetchGroupPolicies } from '../../../redux/actions/policy-actions';
+import { ListLoader } from '../../../presentational-components/shared/loader-placeholders';
+
+const columns = [{ title: 'Name', cellFormatters: [ expandable ]}, 'Description', 'Roles', 'Created', 'Modified' ];
+
+const GroupPolicies = ({ uuid, fetchGroupPolicies, policies, pagination, history }) => {
+  const [ filterValue, setFilterValue ] = useState('');
+
+  const fetchData = (setRows) => {
+    fetchGroupPolicies({ group_uuid: uuid }).then(({ value: { data }}) => setRows(createRows(data, filterValue)));
+  };
+
+  const routes = () => <Fragment>
+    <Route exact path="/policies/add_policy/:id" render={ props => <AddPolicyWizard { ...props }
+      postMethod={ fetchGroupPolicies } /> }/>
+    <Route exact path="/policies/edit/:id" render={ props => <EditPolicy { ...props }
+      postMethod={ fetchGroupPolicies }/> } />
+    <Route exact path="/policies/deny/:id" render={ props => <RemovePolicy { ...props }
+      postMethod={ fetchGroupPolicies }/> } />
+  </Fragment>;
+
+  const actionResolver = (_policyData, { rowIndex }) =>
+    rowIndex % 2 === 1 ? null :
+      [
+        {
+          title: 'Edit',
+          onClick: (_event, _rowId, policy) =>
+            history.push(`/policies/edit/${policy.uuid}`)
+        },
+        {
+          title: 'Delete',
+          style: { color: 'var(--pf-global--danger-color--100)'	},
```

Comment:
can you do the styling via `className`.

And you have bunch of extra spaces before the closing bracket.

Reply:
I do not think the className can be used here - I'll look at this in a future PR - as I have the same styling in all the table kebab menus. There was a tab instead of a space - changed that.